lld/test/ELF/dtlto/imports.test

```diff
@@ -0,0 +1,69 @@
+# REQUIRES: x86
+
+## Check that DTLTO handles imports files correctly.
+
+# RUN: rm -rf %t.dir && split-file %s %t.dir && cd %t.dir
```

Comment:
could just use `%t` instead of `%t.dir` to make the line shorter...

Reply:
Done. Thanks.